msteams-platform/tabs/how-to/authentication/tab-sso-overview.md

```diff
@@ -0,0 +1,172 @@
+
+## SSO in Teams at runtime
+
+Achieve SSO in a tab app by obtaining access token for the Teams app user who's currently logged in. This process involves the tab app, Teams, and Azure AD. During this interaction, the app user must give consent for using identity to obtain the access token.
+
+The following image shows how SSO works when a Teams app user attempts to access the tab app:
+
+:::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/sso-runtime-seqd.png" alt-text="Tab single sign-on SSO diagram" border="false" lightbox="../../../assets/images/authentication/teams-sso-tabs/sso-runtime-seqd.png":::
+
+<!--
+```mermaid
+sequenceDiagram
+    User->>Tab app: Opens Teams app
+    Tab app->>Teams Client: 1. Call getAuthToken()
+    Teams Client->>Sign-in and Consent: 2. Check if consent is required
+    Sign-in and Consent->>Teams Client: Prompt for consent from new user
+    Teams Client->>Azure AD: 3. Request access token from Azure AD
+    Azure AD->>Teams Client: 4. Send access token to Teams Client
+    Teams Client->>Tab app: 5. Respond to getAuthToken() with access token
+    Tab app->>Tab app: 6. Parse access token to give access to user
+```
+-->
+
+| # | Interaction | What's going on |
+| --- | --- | --- |
+| 1 | Tab app -> Teams Client | The tab app makes a JavaScript call to `getAuthToken()`, which tells Teams to obtain an access token. |
+| 2 | Teams Client - > Consent form | If the current app user is using your tab app for the first time, Teams displays request prompt to consent, if it's required. The app user must give consent to Teams for using Teams identity to obtain access token from Azure AD. <br> Alternately, there's a request prompt to handle step-up authentication such as two-factor authentication. |
+| 3 | Teams Client -> Azure AD | Teams requests Azure AD endpoint for the access token for the current app user based on Teams identity. |
+| 4 | Azure AD -> Teams Client | Azure AD sends the access token to the Teams Client. The token is a JSON Web Token (JWT), and it's validation works just like token validation in most standard OAuth flows. Teams caches the token on your behalf so that future calls to `getAuthToken()` return the cached token. |
+| 5 | Teams Client -> Tab app | Teams sends the access token to the tab app as part of the result object returned by the `getAuthToken()` call. |
+| 6 | Tab app | The tab app parses the access token using JavaScript to extract required information, such as the app user's email address. The token returned to the tab app is both an access token and an identity token. |
+
+> [!IMPORTANT]
+> The `getAuthToken()` is valid only for consenting to a limited set of user-level APIs, such as email, profile, offline_access, and OpenId. It isn't used for other Graph scopes such as `User.Read` or `Mail.Read`. For suggested workarounds, see [Extend your app with Microsoft Graph permissions](tab-sso-graph-api.md).
```

Comment:
This is true - and it's also notable that this token is an access token for calling your own back-end API. (ie. the audience claim is set to your app). This allows your app to authenticate the user and authorize access to its own API. If the application doesn't need the Microsoft Graph or other services, that's all that's needed.
Calling the Graph is indeed another topic!